### PR TITLE
Fix pull parameter in build()

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

File: docker/client.py [hunk 1]
```diff
@@ -306,7 +306,7 @@ def attach_socket(self, container, params=None, ws=False):
 
     def build(self, path=None, tag=None, quiet=False, fileobj=None,
               nocache=False, rm=False, stream=False, timeout=None,
-              custom_context=False, encoding=None, pull=True,
+              custom_context=False, encoding=None, pull=False,
```
Comment (glogiotatidis) [inline docker/client.py]: Luckily since this was broken anyway, setting it to False won't break anything. :)

**Comment by glogiotatidis** [comment]
See also https://github.com/docker/docker/issues/13631#issuecomment-107634522

File: docker/client.py [hunk 2]
```diff
@@ -352,6 +352,12 @@ def build(self, path=None, tag=None, quiet=False, fileobj=None,
                 'dockerfile was only introduced in API version 1.17'
             )
 
+        # Docker server 1.6 only supports values 1 and 0 for pull
+        # parameter. This was later fixed to support a wider range of
+        # values, includign true / false.
```
Comment (thaJeztah) [inline docker/client.py]: Typo s/includign/including/

Comment (glogiotatidis) [inline docker/client.py]: Updated

**Comment by glogiotatidis** [comment]
Any updates on this? Since it's a bugfix and not a new feature it blocks a few things.

**Comment by shin-** [comment]
On hold until Docker Engine 1.7 lands.

**Comment by shin-** [comment]
Thank you!

**Pull request merged**